release/release_tests.yaml

```
@@ -977,7 +977,7 @@
 
   run:
     timeout: 600
-    script: jupyter nbconvert --to script --output _test start.ipynb && ipython _test.py
+    script: jupyter nbconvert --to script --output _test start.ipynb && ipython _test.py && python app.py
```

I think this needs to be:

```suggestion
    script: jupyter nbconvert --to script --output _test start.ipynb && ipython _test.py && serve run app:entrypoint && python query.py
```